mitxpro/views.py

```diff
@@ -41,6 +41,20 @@ def index(request, **kwargs):  # pylint: disable=unused-argument
     return render(request, "index.html", context=get_js_settings_context(request))
 
 
+def handler404(request, exception):
+    """404: NOT FOUND ERROR handler"""
+    return render(
+        request, "404.html", context=get_js_settings_context(request), status=404
+    )
```

Comment:
Can we use https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.HttpResponseNotFound ? and `HttpResponseServerError` for server error.

Reply:
Updated